src/main/java/com/google/identitytoolkit/GitkitClient.java

```diff
@@ -176,6 +176,79 @@ public GitkitUser validateTokenInRequest(HttpServletRequest request)
     }
     return null;
   }
+  
+  /**
+   * Verifies the user entered password at Gitkit server.
+   *
+   * @param email The email of the user
+   * @param password The password inputed by the user
+   * @param pendingIdToken The GITKit token for the non-trusted IDP, which is to be confirmed by the user
+   * @param captchaChallenge The captcha challenge
```

Comment:
captchaResponse is only required when there are too many failed password logins for an account. I think it is ok to keep the verifyPassword(String email, String password), since that should work for normal cases.

These edge cases (captcha prompt, pendingIdToken linking) are the reason why we initially did not expose the raw VerifyPassword() API in the client library.